examples/webpack.config.js

```diff
@@ -78,7 +81,7 @@ module.exports = async (env) => {
     };
 
     const plugins = [
-        new CleanWebpackPlugin(['dist']),
```

Comment:
['dist'] argument not needed anymore?

Reply:
Yes, it has different interface and it now cleans dist by default.